_data/stage3.yml

```
@@ -1,88 +1,112 @@
-- title: Legacy RegExp features in JavaScript
-  id: proposal-regexp-legacy-features
-  presented:
-    - date: May&#xA0;2017
-      url: >-
-        https://github.com/tc39/notes/blob/master/meetings/2017-05/may-25.md#15ia-regexp-legacy-features-for-stage-3
-  has_specification: false
-  description: This is a specification draft for the legacy (deprecated) RegExp features in JavaScript, i.e., static properties of the constructor like RegExp.$1 as well as the RegExp.prototype.compile method.
+- id: proposal-regexp-legacy-features
   authors:
     - Claude Pache
   champions:
     - Mark Miller
     - Claude Pache
+  description: >-
+    This is a specification draft for the legacy (deprecated) RegExp features in
+    JavaScript, i.e., static properties of the constructor like RegExp.$1 as
+    well as the RegExp.prototype.compile method.
```

All data here is output from the crawler, but if we fix this once, the changes should persist on subsequent runs.
<br>
```suggestion
    JavaScript, i.e., static properties of the constructor like <code>RegExp.$1</code> as
    well as the <code>RegExp.prototype.compile</code> method.
```